jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -132,31 +137,62 @@ def _subset_selected_changed(self, event={}):
         try:
             self._selected_subset = self._get_region_from_subset(subset_selected)
             self._selected_subset.meta['label'] = subset_selected
+
+            if isinstance(self._selected_subset, CirclePixelRegion):
+                self.bg_annulus_inner_r = self._selected_subset.radius
+            elif isinstance(self._selected_subset, (EllipsePixelRegion, RectanglePixelRegion)):
+                self.bg_annulus_inner_r = max(self._selected_subset.width,
+                                              self._selected_subset.height) * 0.5

```

Comment:
For `RectanglePixelRegion` the circular annulus will intersect the corners, which is not desired (you don't want to measure background from same pixels within the aperture).  For rectangles you want the minimum radius to be to the rectangle corner:

```python
0.5 * np.sqrt(self._selected_subset.width**2 + self._selected_subset.height**2)
```

Reply:
Whooops... good catch. Thanks!